src/v/pandaproxy/schema_registry/protobuf.cc

```diff
@@ -470,47 +471,60 @@ encoding get_encoding(pb::FieldDescriptor::Type type) {
     __builtin_unreachable();
 }
 
+using proto_compatibility_result = raw_compatibility_result;
+
 struct compatibility_checker {
-    bool check_compatible() { return check_compatible(_writer.fd); }
+    proto_compatibility_result check_compatible(std::filesystem::path p) {
+        return check_compatible(_writer.fd, std::move(p));
+    }
 
-    bool check_compatible(const pb::FileDescriptor* writer) {
-        // There must be a compatible reader message for every writer message
+    proto_compatibility_result check_compatible(
+      const pb::FileDescriptor* writer, std::filesystem::path p) {
+        // There must be a compatible reader message for every writer
+        // message
```

Comment:
this doesn't appear in the upstream diff. no clue why 😕

Reply:
I must have formatted the file while resolving the merge conflict which shifted the last word to a new line. I've fixed this now to match the original PR.